main.c

```diff
@@ -75,17 +75,15 @@ static struct {
 	int fd;
 	unsigned int i, lastsep;
 	pid_t pid;
-} info;
+} info, wintitle;
```

Comment:
The `i, lastsep` members aren't used for `wintitle` but they're getting removed in #271 anyways. So I thought I'd merge `info` and `wintitle` together.